contracts/ValidatorsRegistration.sol

```diff
@@ -35,18 +36,19 @@ contract ValidatorsRegistration is IValidatorsRegistration, Ownable {
 
     /// @dev Called by a participant who wishes to register as a validator
 	function registerValidator(bytes4 ip, address orbsAddr, string calldata name, string calldata website, string calldata contact) external {
-		require(!isRegistered(msg.sender), "Validator is already registered");
+		require(!isRegistered(msg.sender), "registerValidator: Validator is already registered");
 		validators[msg.sender].registrationTime = now;
 		_updateValidator(ip, orbsAddr, name, website, contact);
+
+		electionsContract().validatorRegistered(msg.sender);
+
 		emit ValidatorRegistered(msg.sender, ip, orbsAddr, name, website, contact);
```

Comment:
In the previous implementation as well as from a logical point of view the order of these two lines is reversed

Reply:
Right, will move